code/_onclick/hud/screen_objects.dm

```diff
@@ -36,7 +36,7 @@
 	/// If FALSE, this will not be cleared when calling /client/clear_screen()
 	var/clear_with_screen = TRUE
 
-/atom/movable/screen/Destroy()
+/atom/movable/screen/Destroy(force, ...)
```

Comment:
surely the ... is unneeded

Reply:
just copying the default definition forward from /datum/proc/Destroy

Reply:
~~I don't like it it looks weird~~

Also uh, why do we need the force here? I didn't think args worked like this

Reply:
We don't *need* anything here, but we like carrying over args because of named parameters. Remember all those PRs to get Initialize overloads including mapload?

Reply:
oh it's because of named parameters? ok fine.
the ... is weird tho can we agree on that

Reply:
we can agree on that, i think sdmm auto suggests it. maybe Destroy itself is using it for some reason